src/solver/simulation/adequacy.cpp

```diff
@@ -342,7 +342,9 @@ bool Adequacy::year(Progression::Task& progression,
 
         optWriter.addTime(w,
                           pProblemesHebdo[numSpace].tempsResolution1[0],
-                          pProblemesHebdo[numSpace].tempsResolution2[0]);
+                          pProblemesHebdo[numSpace].tempsResolution2[0],
+                          pProblemesHebdo[numSpace].tempsUpdate1[0],
+                          pProblemesHebdo[numSpace].tempsUpdate2[0]);
```

Comment:
UGLY

Reply:
Yes we could add a `struct` to encapsulate `tempsResolution`, `tempsUpdate`, etc.